perf-tests/clusterloader2/cmd/clusterloader.go

```diff
@@ -244,8 +250,11 @@ func main() {
 		klog.Exitf("Cannot create report directory: %v", err)
 	}
 
-	if err = util.LogClusterNodes(mclient.GetClient()); err != nil {
-		klog.Errorf("Nodes info logging error: %v", err)
+	if clusterLoaderConfig.ClusterConfig.RPAccess {
+		// todo: use direct access to RPs w/o proxy
```

Comment:
if, assuming this direct access to the RP won't needed, do we still need this changes in this PR ?

Reply:
in case the proxy kubeconfig is used, it prints out the nodes (for logging purpose) which is the current behavior it tries to keep.